mat64/lq.go

```diff
@@ -35,11 +35,11 @@ func LQ(a *Dense) LQFactor {
 	lq := *a
 
 	lDiag := make([]float64, m)
-	projs := make(Vec, m)
+	projs := NewVector(m, nil)
 
 	// Main loop.
 	for k := 0; k < m; k++ {
-		hh := Vec(lq.RowView(k))[k:]
+		hh := lq.RowView(k)[k:]
 		norm := blas64.Nrm2(len(hh), blas64.Vector{Inc: 1, Data: hh})
```

Comment:
Let's do that later when you add the view methods to *Dense you were suggesting.